eng/pipelines/common/global-build-job.yml

```diff
@@ -25,18 +25,12 @@ jobs:
     variables: ${{ parameters.variables }}
 
     steps:
-    - ${{ if eq(parameters.osGroup, 'OSX') }}:
-      - script: |
-          $(setScriptToEchoAndFailOnNonZero) 
-          brew install pkgconfig icu4c openssl autoconf automake libtool pkg-config python3
-          brew link --force icu4c
-          ln -s /usr/local/opt/openssl/lib/pkgconfig/libcrypto.pc /usr/local/lib/pkgconfig/
-          ln -s /usr/local/opt/openssl/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
-          ln -s /usr/local/opt/openssl/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
-        displayName: Install Build Dependencies
-
     - template: /eng/pipelines/common/clone-checkout-bundle-step.yml
 
+    - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
```

Comment:
Merge with the if statement just below?

Reply:
Not sure it's a good idea, the conditions are the same but they serve different purposes. Also, *Install Build Dependencies* and *Disk Usage before Build* are already long step names.

Reply:
What I mean is just use the same condition. Two different steps. Yaml is indentation based so you can have:

```yml
- ${{ if something }}:
  - script: command
     displayName: first command
  - script: command2
     displayName: second command

- script: command3
   displayName: third command
```

So in this example command and command2 are conditioned to something but command3 is unconditional. So what I mean is to move this step after the disk usage step and share the same if, not merge the two commands in one step.

Reply:
Got it, done in 43a3d87b96323df90d67326f252eb632c56611ab.